src/root/insights.js

```diff
@@ -95,40 +95,40 @@ window.c.root.Insights = ((m, c, h, models, _, I18n) => {
         },
         view: (ctrl) => {
             const project = _.first(ctrl.projectDetails()),
```

Comment:
Let's use m.prop here, and always pass it to child components. This way we can be sure all references to project refer to the same object across nested components.